### PR TITLE
Remove obsolete debug output calls in 23 source files.

### PR DESCRIPTION
**Pull request by ywkaras**
In the src/proxy tree.

### CODE REVIEW

**Comment by ywkaras** [comment]
[approve ci rat]

**Comment by ywkaras** [comment]
[approve ci autest]

File: src/proxy/hdrs/HdrToken.cc [hunk 1]
```diff
@@ -47,7 +51,7 @@
   /ericb
 */
 
-static const char *_hdrtoken_strs[] = {
+const char *_hdrtoken_strs[] = {
```
Comment (zwoop) [inline src/proxy/hdrs/HdrToken.cc:54]: Why eliminate static here ?

File: src/proxy/hdrs/HdrToken.cc [hunk 2]
```diff
@@ -118,7 +122,7 @@ static const char *_hdrtoken_strs[] = {
   // RFC-7932
   "br"};
 
-static HdrTokenTypeBinding _hdrtoken_strs_type_initializers[] = {
+HdrTokenTypeBinding _hdrtoken_strs_type_initializers[] = {
```
Comment (zwoop) [inline src/proxy/hdrs/HdrToken.cc:125]: Why not static ?

File: src/proxy/hdrs/HdrToken.cc [hunk 3]
```diff
@@ -168,7 +172,7 @@ static HdrTokenTypeBinding _hdrtoken_strs_type_initializers[] = {
   {(char *)nullptr,        static_cast<HdrTokenType>(0)},
 };
 
-static HdrTokenFieldInfo _hdrtoken_strs_field_initializers[] = {
+HdrTokenFieldInfo _hdrtoken_strs_field_initializers[] = {
```
Comment (zwoop) [inline src/proxy/hdrs/HdrToken.cc:175]: We hate static ?

File: src/proxy/hdrs/URL.cc [hunk 4]
```diff
@@ -93,9 +93,16 @@ int URL_LEN_MMS;
 int URL_LEN_MMSU;
 int URL_LEN_MMST;
 
+namespace
+{
 // Whether we should implement url_CryptoHash_get() using url_CryptoHash_get_fast(). Note that
 // url_CryptoHash_get_fast() does NOT produce the same result as url_CryptoHash_get_general().
-static int url_hash_method = 0;
+int url_hash_method = 0;
```
Comment (zwoop) [inline src/proxy/hdrs/URL.cc:100]: I miss static

File: src/proxy/http2/HPACK.cc [hunk 5]
```diff
@@ -208,7 +211,7 @@ hpack_field_is_literal(HpackField ftype)
 // Try not to use memcmp(sv, sv) and strncasecmp(sv, sv) because we don't care which value comes first on a dictionary.
 // Return immediately if the lengths of given strings don't match.
 // Also, we noticed with profiling that taking char* and int was more performant than taking std::string_view.
-static inline bool
```
Comment (zwoop) [inline src/proxy/http2/HPACK.cc:211]: Poor static.

**Review by zwoop — APPROVED** [review]

**Comment by cmcfarlen** [comment]
Cherry-picked to v10.0.x

**Pull request merged**